data/views/node.2.0.json

```diff
@@ -1,4 +1,13 @@
 {
+<%
+    var _ = injector.get('_');
```

Comment:
Actually, now the template file lacks the unit-test, so I don't suggest putting too much logic in the template file. It's more difficult to reading the template code than code itself.  So adding either `injector` or lodash into template is not good for me.